src/main/java/org/opensearch/neuralsearch/processor/normalization/ScoreNormalizationFactory.java

```diff
@@ -17,13 +24,38 @@ public class ScoreNormalizationFactory {
 
     public static final ScoreNormalizationTechnique DEFAULT_METHOD = new MinMaxScoreNormalizationTechnique();
 
-    private final Map<String, Function<Map<String, Object>, ScoreNormalizationTechnique>> scoreNormalizationMethodsMap = Map.of(
+    private static final Map<String, Function<Map<String, Object>, ScoreNormalizationTechnique>> scoreNormalizationMethodsMap = Map.of(
```

Comment:
this also needs to be renamed, once it became the static final constant, `SCORE_NORMALIZATION_METHODS`?